src/ResourceManager/DataMigration/Commands.DataMigration/help/New-AzureRmDataMigrationConnectionInfo.md

```diff
@@ -70,53 +70,6 @@ Parameter Sets: (All)
 Aliases: 
 Accepted values: SQL, SQLDB
 
-Required: True
-Position: Named
-Default value: None
-Accept pipeline input: False
-Accept wildcard characters: False
-```
-### -AuthType
```

Comment:
Why are these removed?

Reply:
I regenerated the .md files

Reply:
Oh were these removed in a previous release?

Reply:
They are the same. Reverted the change here.